models/fates/R/write.configs.FATES.R

```diff
@@ -201,143 +196,94 @@ write.config.FATES <- function(defaults, trait.values, settings, run.id){
    # CLM
    #clm.param.default <- system.file("clm5_params.c171117.nc",package="PEcAn.FATES")
    #clm.param.file <- file.path(local.rundir,paste0("clm_params.",run.id,".nc"))
-   clm.param.default <- file.path(refcase,"clm5_params.c171117.nc") # probably need to allow custom param file names here (pecan.xml?)
-   clm.param.file <- file.path(local.rundir,paste0("clm_params.",run.id,".nc"))
-   file.copy(clm.param.default,clm.param.file)
-   clm.param.nc <- ncdf4::nc_open(clm.param.file,write=TRUE)
+   ## Position of parameters file?
```

Comment:
Here I see all the old parameter file stuff being commented out, but not what's replacing it

Reply:
This is something we are working on.  We suppose that the ability to modify the parameter values in fates parameter files with the "prior" distribution set by PEcAN is one of the key functionality of PEcAN workflow. There are also many ways to do it. We have been thinking calling the existing [FATES tools](https://github.com/NGEET/fates/blob/main/tools/modify_fates_paramfile.py). It might also be possible to directly modify the netcdf parameter file using R. We are not sure yet which way would be better. It would be great to learn your opinions.